api/managementpb/checks.proto

```diff
@@ -59,6 +76,12 @@ message ChangeSecurityChecksRequest {
 
 message ChangeSecurityChecksResponse {}
 
+message ChangeSecurityChecksIntervalRequest {
+  ChangeSecurityCheckIntervalParams param = 1;
```

Comment:
Why do we need this `ChangeSecurityCheckIntervalParams`? Why not just add required fields to the `ChangeSecurityChecksIntervalRequest` directly?

Reply:
you are right, I must've copied it from `ChangeSecurityChecks` will fix it